hpo_demo_setup.sh

```diff
@@ -52,6 +52,14 @@ function prereq_check() {
 	## Requires minikube to run the demo benchmark for experiments
 	minikube >/dev/null 2>/dev/null
 	check_err "ERROR: minikube not installed. Required for running benchmark. Check if all other dependencies (php,java11,git,wget,curl,zip,bc,jq) are installed."
+	## Check version of minikube to be <= 1.26.1 for support of Prometheus version 0.8.0
+	minikube_ver=$(minikube version | grep "version" | sed 's/minikube version: v\([0-9]\+\).\([0-9]\+\).\([0-9]\+\).*/\1\2\3/')
+	if [ "$minikube_ver" -gt "1261" ]; then
+		echo "Current minikube version not supported:"; 
+    		echo $(minikube version);
```

Comment:
It would be nice to know what version is supported. I'd suggest a message like
```
ERROR: minikube version not supported
Current Version: ${minikube version}, Supported version: 1.26.1
```

Reply:
updated!